modules/admin_manual/pages/configuration/user/user_management.adoc

```diff
@@ -242,3 +242,64 @@ against quotas.
 
 When a user creates a public link share via URL, and allows uploads, any
 uploaded files count against that user’s quota.
+
+== Moving the User Home
+
+Usually, a users home directory is located in the `data/` directory which location is defined in `config.php` and defaults to `/var/www/owncloud/data`. While in smaller installations, the data directory mounts to a physical drive of the server and in bigger installations the data directory is mounted via NFS, this can be suboptimal when having very big installations with thousands of users and/or users having very big space consumtions. This can result in having high load on a single mount point.
+
+owncloud offers moving the home directory location of single users from the default location to another path outside the data directory to distribute the load on the mount points. You can also move back a users home to the location defined in config.php.
+
+NOTE: As a general advice, moving a users home is the last step after all optimisations of the mount point have been set. Carefully monitor over a period of time the changes made before moving a users home.
+
+NOTE: Consider that these new home mount points also need to be part of a xref:maintenance/backup_and_restore/backup.adoc[backup] and xref:maintenance/backup_and_restore/restore.adoc[restore] plan, if one exists.
```

Comment:
```suggestion
NOTE: Consider that these new home mount points also need to be part of a xref:maintenance/backup_and_restore/backup.adoc[backup] and xref:maintenance/backup_and_restore/restore.adoc[restore] plan.
```

I hope that any site that is big enough to need to use multiple mount points for user data storage will definitely have a backup and restore system!

Reply:
👍